app/controllers/application_controller.rb

```diff
@@ -22,6 +22,17 @@ class ApplicationController < ActionController::Base
   slimmer_template 'wrapper'
 
 protected
+
+  # Set up the navigation helpers. This will make breadcrumbs appear for pages
+  # that have a content item. In the near future this will also provide data to
+  # render the sidebar.
+  def setup_navigation_helpers
+    content_item = content_store.content_item(request.path)
```

Comment:
There won't be a content item for every request path, for example local transactions have many separate [results pages](https://www.gov.uk/pay-council-tax/lambeth) but there's only one content item. [Guides](https://www.gov.uk/adoption-pay-leave/eligibility) and any other mainstream multipart formats also don't have content items for each page. You might be able to use only the first path segment instead, but I don't know whether that would work for all formats.